reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -8,6 +8,11 @@
 
 <header id="liveupdate-header">
   <h1 id="liveupdate-title">${thing.event.title}</h1>
+  % if thing.event.description:
+  <div id="liveupdate-description">
```

Comment:
I think this layout change does not work since description can be customized (vs "one or two sentences").